docs/wip/data-fetching.md

```diff
@@ -1,8 +1,45 @@
-## Gatsby config queries
-
 ## How to fetch data for your pages
 
+One of the main benefits of Gatsby is its hybrid nature capable of static and dynamic data fetching, which means that is possible to fetch data in build time as well in runtime on client-side. The choice of which approach to use depends on the nature of the data and its context, in the section below we will explain the difference with examples.
+
+> If you want to read more about this topic, check the [Data Fetching](https://www.gatsbyjs.com/docs/data-fetching/) Gatsby documentation.
```

Comment:
_If you want to read more about this topic, check the [Data Fetching](https://www.gatsbyjs.com/docs/data-fetching/) Gatsby documentation._ 
 
 Using italic instead of a quote mark